cloudformation/template.yaml

```diff
@@ -0,0 +1,684 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Transform:
+  - AWS::Serverless-2016-10-31
+
+Description: My Dev Portal Stack
+Parameters:
+  ArtifactsS3BucketName:
+    Type: String
+    Description: The S3 bucket in which the Lambda function code is stored. Bucket
+      names are region-unique, so you must change this.
+
+  DevPortalSiteS3BucketName:
+    Type: String
+    Description: The S3 bucket in which the web application code is stored. Bucket
+      names are region-unique, so you must change this.
```

Comment:
Same here as the comment in `base.yaml` about regional vs globally unique bucket names